influxdb/client.py

```diff
@@ -219,7 +235,10 @@ def request(self, url, method='GET', params=None, data=None,
         :raises InfluxDBClientError: if the response code is not the
             same as `expected_response_code` and is not a server error code
         """
-        url = "{0}/{1}".format(self._baseurl, url)
+        if self.__uri == '':
+            url = "{0}/{1}".format(self._baseurl, url)
+        else:
+            url = "{0}/{1}/{2}".format(self._baseurl, self.__uri, url)
```

Comment:
It can be performed at the `self.__baseurl` initialization, your right... when reading the code first time, I focused on the `url` string creation.